WEB-INF/src/hibernate.cfg.xml

```diff
@@ -13,10 +13,13 @@
         <!--<property name="connection.password">PASSWORD</property>-->
         <property name="dialect">org.hibernate.dialect.PostgreSQLDialect</property>
         <property name="show_sql">false</property>
-        <property name="hibernate.jdbc.batch_size">100</property>
+        <property name="hibernate.jdbc.batch_size">1</property>
```

Comment:
this can be set back to 100 (i changed it for bebugging)

Reply:
Thanks, have changed this back